### PR TITLE
feat: custom AA test data

### PR DESCRIPTION
**Pull request by Meyanis95**
## Motivation

This PR is adding scripts to generate custom test data from the test data provided by UIDAI.
- Add version specifier at the beginning
- Add the last 4 digits of the phone number after VTC
- Update signing timestamp
- Create new RSA key pairs

## Change Summary

- Add `generateTestData.ts`:
  - Add version specifier at the beginning
  - Add the last 4 digits of the phone number after VTC
  - Update signing timestamp
  - Sign the newly generated data with our own RSA key pair
  - Store the new QR value in a JSON alongside ID fields
- Add `generateTestKeys.ts`, script to generate new RSA key pairs and store the corresponding certificates
- Fix `/circuits` tests
- Helper function `returnFullId()`, to return all the ID fields
- Fixed `extractPhoto()` helper function, using [JP2000](https://www.npmjs.com/package/jpeg2000) to check if bytes are the right ones.

The only parts not working, for now, are the photo extraction test and the test on `userNullifier`.
The issue with the photo is that in the circuit we elevate all the bytes equal to 255 to `n` which is the index of 255 bytes among all the 255 bytes.
@saleel I tried some workarounds but did not find a solution to modify the circuits so we can solve this issue. We can talk about it when you're back if that works for you.

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [x] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [ ] PR has a [change set](https://github.com/privacy-scaling-explorations/anon-aadhaar/CHANGELOG.md)
- [x] PR has been tagged with a change label(s) (i.e. documentation, feature, bug fix, or chore)
- [x] PR includes documentation if necessary.

### CODE REVIEW

**Review by saleel — APPROVED** [review]

**Comment by saleel** [comment]
React tests are failing, but I think that is yet to be updated with new test data?
I will merge and continue changes on `develop`

File: packages/circuits/src/helpers/extractor.circom [hunk 1]
```diff
@@ -261,6 +261,6 @@ template QRDataExtractor(maxDataLength) {
     component photoExtractor = PhotoExtractor(maxDataLength);
     photoExtractor.nDelimitedData <== nDelimitedData;
     photoExtractor.startDelimiterIndex <== delimiterIndices[photoPosition() - 1];
-    photoExtractor.endIndex <== nonPaddedDataLength - 1;
+    photoExtractor.endIndex <== nonPaddedDataLength - 65;
```
Comment (saleel) [inline packages/circuits/src/helpers/extractor.circom:264]: `nonPaddedDataLength` was expected to be the length without the signature. Either works though.

Comment (Meyanis95) [reply packages/circuits/src/helpers/extractor.circom:264]: Here the 65 bytes removed are not the bytes of the signature but the bytes of the phone hash and the email hash.
Also, it should be dynamic and conditioned on the is_mobile_or_phone variable.

Comment (saleel) [reply packages/circuits/src/helpers/extractor.circom:264]: Oh ok. But email and mobile hash was not part of the V2 data when I checked. Let me check again.

**Pull request merged**